test-pad/test-pad

```diff
@@ -0,0 +1,313 @@
+#!/bin/sh
+
+. "../bin/libstd-ecm.sh"
+
+usage() {
+    echo "usage: $0 [-r repo] [-c cluster] [-v version | -b location ] [sbdh]
+    -r    repository (k3s/rke2)
+    -c    cluster type
+    
+    -v    k3s/rke2 version to use
+    OR
+    -b    location of local k3s/rke2 binary to use
+
+    -k    kill cluster
+    -d    kill cluster and delete files
+    -s    skip Vagrantfile download
+    -o    only download Vagrantfile, don't create cluster
+    -h    show help
+
+cluster types:
+    - basic:        2 VMs, 1 server, 1 agent
+    - basic-lite:   1 VM,  1 server
+    - ha:           5 VMs, 3 servers, 2 agents
+    - ha-lite:      3 VMs, 3 servers
+    - split:        7 VMs, 3 etcd-only server, 2 cp-only servers, 2 agents. Taints on etcd and control-plane
+    - split-lite:   3 VMs, 1 etcd-only server, 1 cp-only servers, 1 agent. Taints on etcd and control-plane
+    - rancher:      4 VMs, 1 single server with rancher, 3 blank VMs ready for provisioning
+examples:
+    $0 -r k3s -v v1.22.9+k3s1 -c basic 
+    $0 -r k3s -v v1.23.5+k3s1 -c ha
+    $0 -r k3s -c ha-lite -b ../../k3s/dist/artifacts/k3s
+    "
+}
+
+# TODO - dual-stack:   3 VMs, 3 servers with IPv4 and IPv6
+
+install_binary() {
+    if [ "$BINARY_LOCATION" ]; then
```

Comment:
Can you add braces to all variables in this script? I'd like to keep everything as consistent as possible.

Reply:
About to add alot of braces.